github-dark.css

```diff
@@ -1422,7 +1422,8 @@
   .fullscreen-overlay textarea, .avatar-child, .discussion-item .label-color,
   .coupons .coupon-form-body, .label-private, .repo-file-upload-progress,
   .more-repos, .header .header-search-wrapper.focus,
-  .blame-commit .blame-commit-info, .label, .Label, .project-card {
+  .blame-commit .blame-commit-info, .label, .Label, .project-card,
+  .ajax-pagination-form .ajax-pagination-btn:focus {
```

Comment:
lol, so can we use a NON base color, is there something you suggest or just leave as is?

Reply:
OK I tried ALL the existing box shadows, none worked so added this.

```css
 .ajax-pagination-form .ajax-pagination-btn:focus {
    box-shadow: 0 0px 2px #777 !important;
  }
```

I feel a base-color box-shadow on a black/grey button looks weird, so I had to give this a try :)

![popover](https://user-images.githubusercontent.com/31389848/42412853-59799ed4-820c-11e8-9963-a31d6e040e06.gif)

So whenever you have time and can focus on this, Ill either add the css above or remove the selector as you said.

Reply:
Well removing the selector maintains the current focus coloring - it's blue when you turn GHD off. Personally, I don't really care since the button highlights when focused or active.

Reply:
> it's blue when you turn GHD off.

:focus here is background-color: #f6f8fa; when GHD off and its weird border-shadow which comes from `.btn-outline:focus { border-color: #0366d6;	box-shadow: 0 0 0 0.2em rgba(3,102,214,0.4); }`

This reminds me a bit of the other #653 issue.

It looks more natural to me with a grey box-shadow, so lets ask @silverwind for his opinion... lol